pkg/services/folder/folderimpl/sqlstore.go

```diff
@@ -344,27 +346,74 @@ func (ss *sqlStore) GetHeight(ctx context.Context, foldrUID string, orgID int64,
 	return height, nil
 }
 
-func (ss *sqlStore) GetFolders(ctx context.Context, orgID int64, uids []string) ([]*folder.Folder, error) {
-	if len(uids) == 0 {
+// GetFolders returns org folders by their UIDs.
+// If setFullpath is true it computes also the full path of a folder.
+// The full path is a string that contains the titles of all parent folders separated by a slash.
+// For example, if the folder structure is:
+//
+//	A
+//	└── B
+//	    └── C
+//
+// The full path of C is "A/B/C".
+// The full path of B is "A/B".
+// The full path of A is "A".
+// If a folder contains a slash in its title, it is escaped with a backslash.
+// For example, if the folder structure is:
+//
+//	A
+//	└── B/C
+//
+// The full path of C is "A/B\/C".
+func (ss *sqlStore) GetFolders(ctx context.Context, q folder.GetFoldersQuery) ([]*folder.Folder, error) {
+	if len(q.UIDs) == 0 {
```

Comment:
I think this is counterintuitive. The method name implies that it returns folders, and it is the first method I would use if I need all folders. This filter is more appropriate for GetFoldersByUIDs.